virtcontainers/filesystem.go

```diff
@@ -112,6 +118,11 @@ type resourceStorage interface {
 	// for the actual resource and the URI base.
 	containerURI(sandboxID, containerID string, resource sandboxResource) (string, string, error)
 	sandboxURI(sandboxID string, resource sandboxResource) (string, string, error)
+	sandboxesURI(resource sandboxResource) (string, string, error)
```

Comment:
The interface naming is really confusing. How about `globalSandboxURI`?

Reply:
Yes sounds good.

Reply:
done